technology_and_tooling/best_practices/code_style_python.md

```diff
@@ -2,6 +2,15 @@
 name: Code Style
 dependsOn: []
 tags: [python]
+learningOutcomes:
+  - Understand the benefits of following community coding conventions.
+attribution:
+  - citation: >
+      This material was edited from the original in "Intermediate Research Software
+      Development Skills" hosted by the Carpentries Incubator.
+    url: https://carpentries-incubator.github.io/python-intermediate-development/
+    image: https://carpentries-incubator.org/assets/img/carpentries-incubator.svg
+    license: CC-BY-4.0
 ---
 
 :::callout
```

Comment:
can you please also remove this callout (its a remnant from before we had the attribution metadata)

Reply:
Thanks for highlighting this @martinjrobins, this has been fixed in 9b68c01.